CollAction/Controllers/AdminController.cs

```diff
@@ -250,7 +250,7 @@ public IActionResult Index()
                         "<br>" +
                         "The deadline of the project you have started on www.collaction.org has passed. We're very happy to see that the target you have set has been reached! Congratulations! Now it's time to act collectively!<br>" +
                         "<br>" +
-                        "The CollAction Team might reach out to you with more specifics (this is an automated message). If you have any further questions yourself, feel free to contact the CollAction Team at collactionteam@gmail.com. And don’t forget to tag CollAction in your messages on social media so we can help you spread the word on your achievement (FB: @collaction.org, Twitter: @collaction_org)!<br>" +
+                        "The CollAction Team might reach out to you with more specifics (this is an automated message). If you have any further questions yourself, feel free to contact the CollAction Team at collactionteam@gmail.com. And don’t forget to tag CollAction in your messages on social media so we can help you spread the word on your achievement ( FB: <a href='https://www.facebook.com/collaction.org/'>@collaction.org</a>, Twitter: @collaction_org)!<br>" +
```

Comment:
I think you added one space too much (looking at CA-318 on JIRA).

There's a space added between FB and (: "( FB" that probably shouldn't be there.

Reply:
This is fixed in 8458770